clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/ingestion/heartbeat/HeartbeatMonitoringService.java

```diff
@@ -222,10 +222,12 @@ protected void recordLags(
   protected void record() {
     recordLags(
         leaderHeartbeatTimeStamps,
-        ((storeName, version, region, lag) -> versionStatsReporter.recordLeaderLag(storeName, version, region, lag)));
+        ((storeName, version, region, lag) -> versionStatsReporter
+            .recordLeaderLag(storeName, version, region, System.currentTimeMillis() - lag)));
     recordLags(
         followerHeartbeatTimeStamps,
-        ((storeName, version, region, lag) -> versionStatsReporter.recordFollowerLag(storeName, version, region, lag)));
+        ((storeName, version, region, lag) -> versionStatsReporter
+            .recordFollowerLag(storeName, version, region, System.currentTimeMillis() - lag)));
```

Comment:
Can we just pass start time here and then do subtraction (to get elapsed time) in `HeartbeatStat::recordLeaderLag` & HeartbeatStat::recordFollowerLag`.  This way we don't need to invoke `System.currentTimeMillis()` twice for each call..?